hdk/cl/developer_designs/cl_firesim/build/scripts/create_dcp_from_cl.tcl

```diff
@@ -213,8 +213,9 @@ switch $strategy {
 #set phys_opt 0
 set phys_opt 1
 
-puts "AWS FPGA: ([clock format [clock seconds] -format %T]) Calling the encrypt.tcl.";
-
+#puts "AWS FPGA: ([clock format [clock seconds] -format %T]) Calling the encrypt.tcl.";
+# FireSim: note, this only moves sources into the encrypt directory without
+# actually encrypting them. Modify encrypt.tcl to renable CL-source encryption.
```

Comment:
```suggestion
# FireSim: note, this only moves sources into the encrypt directory without actually encrypting them.
# See comment in encrypt.tcl for more detail.
```

When I read `Modify encrypt.tcl to reenable CL-source encryption.` I almost wonder if that is a TODO type statement that needs to happen for some reason.  I feel like it is confusing and doesn't need to be said.  Having a comment about encrypt.tcl being a misnomer is a good idea though.